### ISSUE TITLE
Add more formats

### ISSUE THREAD

**Issue by dseomn** [performous/performous#825]
### Suggestion

Some song formats I have in my collection that aren't supported yet, I think:
* dwi, e.g., from https://search.stepmaniaonline.net/pack/id/2025. I couldn't find a spec with a quick search, but StepMania has a parser: https://github.com/stepmania/stepmania/blob/5_1-new/src/NotesLoaderDWI.cpp
* [ssc](https://github.com/stepmania/stepmania/wiki/ssc), e.g., from https://www.clubfantastic.com/#download

And audio formats:
* [opus](https://en.wikipedia.org/wiki/Opus_(audio_format)), e.g., from https://projectoutfox.com/outfox-serenity/volume-i

### Extra info/examples/attachments

If I'm understanding the code correctly, these are the lists of song and audio formats that are already supported, right?
https://github.com/performous/performous/blob/baab998042eebc17a352d4542803108eff31a6b7/game/songs.cc#L255
https://github.com/performous/performous/blob/baab998042eebc17a352d4542803108eff31a6b7/game/songparser.cc#L136-L145

**Comment by Baklap4** [comment]
> If I'm understanding the code correctly, these are the lists of song and audio formats that are already supported, right?

You're correct!
Would be nice to see a PR following this

**Comment by dseomn** [comment]
>  Would be nice to see a PR following this

Yup. This isn't that high of a priority for me at the moment, but I might get around to it at some point if that changes.

**Comment by taynaron** [comment]
opus support added, the rest is still to-do.
